### ISSUE TITLE
Is it possible to have a larger, bolder title?

### ISSUE THREAD

**Issue by gitnoob23** [SeregPie/THREE.TextSprite#27]
Hello,

Wondering if it is possible to link two text sprites together, so one could be a left justified, larger and bold title. Then directly under it would be a multi-line left justified, smaller and normal body of text.

I was attempting to do this but was having issues with the X placement. Even making them both the same X position, the title wouldn't line up with the body.

Something like this for example:

**THIS IS A TITLE PRETEND IT IS LARGER**
This is a sentence in this paragraph.
This is another sentence in this paragraph.
Everything is left justified. :)

Thoughts?

Thanks!

**Comment by SeregPie** [comment]
I think this should be possible with this [`THREE.TextPlane`](https://github.com/SeregPie/THREE.TextPlane).
It's not a sprite. It's a mesh. So it should be easier to adjust the position.

**Comment by gitnoob23** [comment]
Thank you for the suggestion!

So I have implemented textplane. But it seems to be giving me the same issue.

I have two text planes, one is the title and one is the body of text. There doesn't seem to be a way to make sure they are snapped together and both left justified. Unless I am missing something? I have also tried to get their widths since they are both different and then I was going to reposition the title based on the width of the body, but the width always returns 1, even though the body is much wider than the title.

Thoughts?
